### DISCUSSION TITLE
App FAQ: Register "scheduled task" via the app

### DISCUSSION THREAD

**Discussion by cuonghuynh** [shopware/platform#3418] [App FAQ] [answered]
### What kind of question do you have about the app system?

General Setup

### Describe your question or feedback

I would like the shops to trigger the webhook in my app at the scheduled time

Is there a way to register a "scheduled task" via the App?

**Comment by shyim** [comment, marked as answer]
Hey,

It's right now not possible to register scheduled tasks inside Shopware for apps. Our preferred way would be that you register a cronjob / similar task on your app server and then do what you want.
